src/pages/components/select/usage.mdx

```diff
@@ -141,59 +181,187 @@ Inline selects have less visual weight on a page because they are borderless.
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| _Enabled_   | When the select is live and a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component. An enabled select field should contain a default value.                                                |
+| _Hover_     | When a user's mouse cursor is hovering over the field.                                                                                                                                                                                                       |
+| _Focus_     | When a user tabs to or clicks on the select field, the field becomes focused, indicating the user has successfully navigated to the component.                                                                                                               |
+| _Error_     | When the required select value has not been selected. It can also be triggered due to a system error. This state requires a user response before data can be submitted or saved.                                                                             |
+| _Warning_   | When you need to call the user's attention to an exception condition. The condition might not be an error but can cause problems if not resolved.                                                                                                            |
+| _Disabled_  | When the user is not allowed to interact with the select due to either permissions, dependencies, or prerequisites. The disabled state completely removes the interactive function from a component. The styling is not subject to WCAG contrast compliance. |
+| _Skeleton_  | Used on an initial page load to indicate that the select has not yet fully loaded.                                                                                                                                                                           |
+| _Read-only_ | _Coming soon!_                                                                                                                                                                                                                                               |
+
+<Row>
+<Column colLg={12}>
+
+![Select interactive states](images/select-usage-9.png)
+
+</Column>
+</Row>
+
+### Interactions
+
+#### Mouse
+
+Users trigger a dropdown menu to open by clicking the chevron icon or clicking
+anywhere within the field. Users can close the menu by clicking the chevron icon
+or clicking outside of the menu. To select an option, the user can click
+anywhere inside an option container.
+
+<Row>
+<Column colLg={8}>
+
+![Select mouse interaction](images/select-usage-10.png)
+
+</Column>
+</Row>
+
+#### Keyboard
```

Comment:
There is another code bug happening here.
<img width="1013" alt="Screen Shot 2022-12-12 at 10 52 53 AM" src="https://user-images.githubusercontent.com/43969356/207091145-ccf7d663-8d7c-4f2b-a9dc-02ee5dfbfb03.png">